src/components/CardSeparator.tsx

```diff
@@ -10,7 +11,27 @@ import classnames, {
   width,
 } from 'classnames/tailwind'
 
-type Color = 'accent' | 'primary' | 'secondary' | 'tertiary' | 'transparent'
+type Gradient =
+  | 'secondary-to-transparent'
+  | 'accent-to-secondary'
+  | 'accent-to-transparent'
+interface CardSeparatorProps {
+  number: number
+  gradient: Gradient
+  vertical?: boolean
+  customHeight?: THeight
+}
+
+function getTailwindClassnames(gradient: Gradient) {
+  switch (gradient) {
+    case 'secondary-to-transparent':
+      return 'from-secondary'
+    case 'accent-to-secondary':
+      return { 'from-accent': true, 'to-secondary': true }
```

Comment:
And this is a way to use two tailwind properties, we can't use an array unfortunately, only `{ prop: true, prop2: true}`. We did it previously in some places, it lookes like:
```TS
gradient(color === 'green' ? { 'from-blue': true, 'to-green': true } : undefined)
```